interpreters/src/drop.rs

```diff
@@ -108,12 +109,15 @@ impl<C: Manager> DropInterpreter<C> {
             table_engine: self.table_engine,
         };
 
-        let dropped = schema
+        if !schema
             .drop_table(request, opts)
             .await
-            .context(SchemaDropTable { table: &table })?;
+            .context(SchemaDropTable { table: &table })?
+        {
+            info!("Failed to drop table {}", &table);
```

Comment:
The returned false indicates real drop does not happen (usually the table is dropped by another requests) so the log message may read:
```suggestion
            warn!("Table {} has been dropped already", &table);
```